.github/README.md

```diff
@@ -39,7 +39,23 @@ Streamlit Analytics2 is a maintained, powerful tool for tracking user interactio
 
 ## Contributing
 
-We're actively seeking additional maintainers to help improve Streamlit Analytics2. If you're interested in contributing, please check out our [Contributing Guidelines](.github/CONTRIBUTING.md) to get started. We welcome pull requests, bug reports, feature requests, and any other feedback.
+### Session-Level Analytics
+
+We have added a new feature to track session-level analytics. The default behavior of tracking across all sessions of your app is still supported, but now we also enable tracking at the individual session level as well. To do this, simply add a string representing a unique session ID to the track() function (recommended to be a UUID4):
+
+```python
+import streamlit as st
+import streamlit_analytics2 as streamlit_analytics
+
+with streamlit_analytics.track(session_id="1234567890"):
```

Comment:
Some thoughts:
- What would be a good value to set for session_id? 
- Could it be a random value or does it matter?
- Does this preserve user privacy? Are end users of an app required to provide consent to having their session state inputs captured?
Please share your thoughts and if I have understood the purpose of session_id properly

Reply:
* I mention this in the README, but I use `str(uuid4())` personally, to try and quasi-guarantee uniqueness across sessions
    * RE: randomness, certainly can be, but again the app dev would need to make sure they aren't providing an already-used session_id or their analytics will get pushed together for two real sessions into what looks like a single session (since firestore logic simply does a document update in that scenario). I essentially tested this on accident by virtue of starting and stopping the server but retaining the same test session_id and it simply updates the analytics doc it created.
* Right now we use this in our downstream app by using an `st.dialog()` call to pop up a cookie accept/reject modal that then determines if we can use cookie tracking or not. If the user rejects cookies, the session_id is simply set to None. So the TLDR; here is that we expect the user of SA2 to handle that stuff for the sake of abstraction. Perhaps the easiest solution here is to provide this example logic (happy to also provide the code for the simple st.dialog call here) in the Wiki?

```python
@st.dialog("Cookie Notice", width="large")
def deploy_cookie_banner():
    # Remove the Close/X button so user must decide to accept or reject
    st.html(
        """
        <style>
            div[aria-label="dialog"]>button[aria-label="Close"] {
                display: none;
            }
        </style>
    """
    )
    st.markdown(
        dedent(
            """
        We use cookies to enhance your experience and analyze site usage. 
        By clicking "Accept", you consent to our use of cookies.
        """
        )
    )

    accept = st.button("Accept all cookies", type="primary", use_container_width=True)
    reject = st.button("Use only strictly necessary cookies", use_container_width=True)

    # TODO: add a privacy policy link
    # st.markdown("[Learn More](https://example.com/privacy)", help="Read our privacy policy")

    if accept:
        ss.cookies_accepted = True
        st.rerun()
    elif reject:
        ss.cookies_accepted = False
        st.rerun()
```

Reply:
Perfect, that makes sound sense  
We will provide documentation on how to respect user privacy and in time, perhaps just provide the cookie button flow ourselves